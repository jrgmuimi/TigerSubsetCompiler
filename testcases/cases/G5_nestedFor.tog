let
    var for_ctrl1 : int
    var for_ctrl2 : int
    var random    : int
    var num : int
in
    for for_ctrl1:=0 to 11 do
        num := 0;
        for for_ctrl2:=1 to for_ctrl1+1 do
            num := num + for_ctrl2;
            for random:=0 to (for_ctrl1*for_ctrl2*10) do
                random := random + 7            // extra +7 to make the loop ends faster. It doesnt do anything
            end
        end;
        printint(num);
        printstring("\n")
    end
end
        