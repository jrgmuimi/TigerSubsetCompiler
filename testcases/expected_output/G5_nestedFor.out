MARS 4.5  Copyright 2003-2014 Pete Sanderson and Kenneth Vollmar

1
3
6
10
15
21
28
36
45
55
66
78

